Examples/Scripts/Python/propagation.py

```diff
@@ -18,43 +24,63 @@ def runPropagation(trackingGeometry, field, outputDir, s=None, decorators=[]):
 
     rnd = acts.examples.RandomNumbers(seed=42)
 
+    addParticleGun(
+        s,
+        ParticleConfig(num=1000, pdg=acts.PdgParticle.eMuon, randomizeCharge=True),
+        EtaConfig(-4.0, 4.0),
+        MomentumConfig(1 * u.GeV, 100 * u.GeV, transverse=True),
+        rnd=rnd,
+    )
+
+    # Run particle smearing
+    trackParametersGenerator = acts.examples.ParticleSmearing(
+        level=acts.logging.INFO,
+        inputParticles="particles_input",
+        outputTrackParameters="start_parameters",
+        randomNumbers=rnd,
+        sigmaD0=0.0,
+        sigmaZ0=0.0,
+        sigmaPhi=0.0,
+        sigmaTheta=0.0,
+        sigmaPRel=0.0,
+        addCovariances=False,
+    )
+    s.addAlgorithm(trackParametersGenerator)
+
     nav = acts.Navigator(trackingGeometry=trackingGeometry)
 
     stepper = acts.EigenStepper(field)
     # stepper = acts.AtlasStepper(field)
     # stepper = acts.StraightLineStepper()
-    # stepper = acts.SympyStepper(field)
+    #  stepper = acts.SympyStepper(field)
```

Comment:
nitpick: unrelated change

Reply:
Correct, it needs #3433 anyway, will change.